orttraining/orttraining/test/python/orttraining_run_bert_pretrain.py

```diff
@@ -317,6 +335,14 @@ def setup_training(args):
     logger.info("setup_training: args.train_batch_size = %d", args.train_batch_size)
     return device, args
 
+def setup_torch_distributed(world_rank, world_size):
+    os.environ['RANK'] = str(world_rank)
+    os.environ['WORLD_SIZE'] = str(world_size)
+    os.environ['MASTER_ADDR'] = 'localhost'
+    os.environ['MASTER_PORT'] = str('12345')
+    torch.distributed.init_process_group(backend='nccl', world_size=world_size,
```

Comment:
Is `torch.distributed` used for anything else than creating barriers to coordinate ranks for checkpointing API?

Reply:
It is only used for creating barriers so that a single test can execute the entire workflow.